src/main/java/com/ixortalk/organization/api/rest/OrganizationAdminController.java

```diff
@@ -95,7 +95,7 @@
     private Optional<User> getUser(Long organizationId, Long userId) {
         return userRestResource.findById(userId).map(user -> {
             Organization organization = organizationRestResource.findById(organizationId).orElseThrow(ResourceNotFoundException::new);
-            if (!organization.containsUser(user) || !auth0Users.userExists(user.getLogin())){
```

Comment:
`auth0Users` field can be removed too then? Such that we don't have any Auth0 reference anymore in this controller.